mcsema/BC/Function.cpp

```diff
@@ -166,6 +167,34 @@ static void InlineSubFuncCall(llvm::BasicBlock *block,
   (void) new llvm::StoreInst(call, mem_ptr, block);
 }
 
+// Add a invoke to another function and then update the memory pointer with the
+// result of the function
```

Comment:
Add a bit more documentation, saying that we need to stash the stack and frame pointers. Maybe one other thing we could do, though, is to modify Remill's `__remill_basic_block` and add two more variables: `STACK_POINTER` and `FRAME_POINTER`. Then we can just find those variables in the function using the normal means, and it'd be arch-independent. I might prefer this mechanism, in fact.